zhaquirks/lixee/zlinky.py

```diff
@@ -170,6 +178,7 @@ class ZLinkyTIC(CustomDevice):
                 DEVICE_TYPE: zha.DeviceType.METER_INTERFACE,
                 INPUT_CLUSTERS: [
                     Basic.cluster_id,
+                    PowerConfiguration.cluster_id,
```

Comment:
Adding the PowerConfiguration cluster for both versions of the quirk allows supporting that cluster for devices that were already existing but upgraded from pre-v12 to v12 without having to delete then re-add the device from ZHA.